### PR TITLE
Update pre deployment calibration procedure section

### LINKED ISSUE #144: correct/better organising of optode commands, firmware versions
@s-pearce

"In section 3.4.1.2 # 3 it says to "Do Stop" is the command to stop the Aanderaa 4831/4330 from sampling and to use "CR + LF" in the terminal. This is a bit of a mixed message. In optode firmware versions smaller than 3.x.x (Framework 3), it did not require the "CR + LF", only the "CR" and the command to stop was "Do Stop". In 3.x.x and higher (as of 2021, in 5.x.x framework), the optode does require "CR+LF" and the command to stop was changed to only a "Stop", no longer requiring the "Do".
This is described in Appendix 11 "Product Change Notification: Framework 3" in the Aanderaa 4330/4835/4831 Optode manual. Anyway, nobody is likely to use firmware less than 3.x.x anymore, so I suggest that the "Do Stop" change to "Stop" above.
Also, I believe it was framework 3 that first allowed the SVU/Multipoint-calibration, and firmware less than 3.x.x only used the Mk2 equation.
-Stuart Pearce"

### PR DESCRIPTION
**Pull request by tomhull**
Rewrite of the pre-deployment calibration steps and various smaller fixes. Should close #144 

@patricialg one bit I don't understand and I think still needs review (I can open a issue if needed) is the bit about calibration in the ballast tank using a sonde. Surely this doesn't help because now you need to calibrate the sonde as well?

### CODE REVIEW

**Comment by patricialg** [comment]
HI @tomhull ,
Yo are right about rewriting that. We used the sonde because we were calibrating it also because it was going the next day to the sea with the glider for the deployment (so it's not really necessary in all cases or you can decide to calibrate it another time). It also help to check if something is too wrong with glider or sonde sensors because you have both measuring at the same time ;) 
What do you think?
Cheers,
Patry

**Comment by soerenthomsen** [comment]
I agree with Tom in regards of the sonde in the tank and think it should not be a general recommendation.

File: sections/oxygen_pre_deployment.md [hunk 1]
```diff
@@ -39,173 +39,109 @@ When applying the tape, be sure that it is not in contact with any other metal p
...
 4. Once the glider is in the ballasting tanks, place the silicon tube for sampling near the sensor.
-5. Once the sensor measurements are stable (variations in the measurements are not higher than the precision/resolution of the sensor), start collecting Winkler samples.  Take samples every 5-10 minutes, in total 4-6 samples will be required. *NOTE: Record the time we collect a sample for Winkler titration.*
-6. A multiparameter sonde with a DO sensor whose precision is less than 0.1 % can be measuring in the tank near the glider’s sensors (record at least values for Temperature, Conductivity and DO). 
+5. Once the sensor measurements are stable (variations in the measurements are not higher than the precision/resolution of the sensor), start sampling water for Winkler analysis. Take paired samples every 5-10 minutes and record the time taken. 4 to 6 samples should be taken in total.
+6. A multiparameter sonde with a oxygen sensor whose precision is less than 0.1 % can be measured in the tank near the glider’s sensors (record at least values for Temperature, Conductivity and DO).
```
Comment (soerenthomsen) [inline sections/oxygen_pre_deployment.md:134]: should we take this out? I think to introduce another instrument which has to be calibrated too to be helpful just creates more work or?

**Review by soerenthomsen — APPROVED** [review]
There is just the multi parameter sonde comment left and after I think this can be merged @tomhull

**Review by soerenthomsen — APPROVED** [review]

File: sections/oxygen_pre_deployment.md [hunk 2]
```diff
@@ -39,173 +39,107 @@ When applying the tape, be sure that it is not in contact with any other metal p
...
-            Set Passkey(1000)
-            
-            Set Interval(1)
-            
-            Save
-12. Type Get All command for saving the final sensor configuration for reference.
-
 ### In situ intercomparison in the tank during ballasting
-This is an extra in situ intercomparison to carry out if  access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before) .
+This is an extra in situ intercomparison to carry out if access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before) (reference PLOCAN).
```
Comment (patricialg) [inline sections/oxygen_pre_deployment.md:127]: @tomhull  Please, can you remove the reference to PLOCAN? we haven't received the doc so there is nothing now

Comment (soerenthomsen) [inline sections/oxygen_pre_deployment.md:127]:
```suggestion
This is an extra in situ intercomparison to carry out if access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before).
```

Comment (soerenthomsen) [reply sections/oxygen_pre_deployment.md]: I removed it.

**Comment by soerenthomsen** [comment]
@tomhull and @patricialg can you both coordinate to finish and merge this PR?

**Comment by patricialg** [comment]
I'm happy with the changes Tom did :)

**Comment by soerenthomsen** [comment]
Did we agree on the "the sonde in the tank" question? @tomhull @patricialg

**Comment by patricialg** [comment]
It was removed from the text since we all agreed that it's not something to consider as general recomendation. I checked and I couldn't find it in the new version. I'll check in cae appears in another place..

**Comment by soerenthomsen** [comment]
Ok perfect! Glad to see that we are closing a lot now! Soon we can submit :)

**Pull request merged**